activerecord/lib/active_record/result.rb

```diff
@@ -43,6 +43,11 @@ def initialize(columns, rows, column_types = {})
       @column_types = column_types
     end
 
+    # Does this result set include the column named +name+ ?
```

Comment:
To follow the same convention on this same class we should do:

```ruby
# Returns true if this result set include the column named +name+
```

Reply:
@rafaelfranca how about `Returns truthy if this result set include the column named +name+`.  I don't want to guarantee the return value is `true` or `false` if we decide to change implementation (I remember there was some controversy because we changed a method to return a truthy value). @fxn thoughts?

Reply:
`true` in our guideline means truthy. `+true+` means the instance of `TrueClass`.

http://guides.rubyonrails.org/api_documentation_guidelines.html#booleans

Reply:
I am not really happy with our convention (that I defined myself many years ago), which is to use fixed-width font for the singletons, and regular font to mean "true" in the Ruby sense of the word (i.e., "truthy"). My thoughts by then were that Ruby defines what is true and false, not what is truthy and falsey, so I stuck with that jargon for respect.

In retrospect, that choice was too subtle and time has shown it doesn't communicate well in practice. It has also the problem that in sample code like

```ruby
user.blank? # => true
```

You cannot distinguish if that is regular or fixed-width, therefore people assume the singleton `true` sometimes and that has made us change some return contracts, to be stricter.

I suggested to change this convention in Basecamp a while back, but it was agreed to leave it like it is.